src/main/java/core/basesyntax/Constants.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public final class Constants {
+    public static final int COLORS = Color.values().length;
+    public static final int SHAPES = Shape.values().length;
```

Comment:
Good names for them would be: COLORS_LENGTH and SHAPES_LENGTH. But it's better not create separate constants for it